test/plugin/scenarios/webflux-scenario/webflux-projectB-scenario/src/main/java/test/apache/skywalking/apm/testcase/sc/webflux/projectB/controller/TestAnnotationController.java

```diff
@@ -59,4 +66,18 @@ public Mono<String> urlPattern(@PathVariable("test") String var) {
     public Mono<String> hello(@RequestBody(required = false) String body) {
         return Mono.just("Hello World");
     }
+
+    @RequestMapping("/testcase/success")
+    public String downstreamMock() {
+        return "1";
+    }
+
+    @GetMapping("/testcase/annotation/mono/onnext")
+    public Mono<String> monoOnNext(@RequestBody(required = false) String body) {
+        return Mono.subscriberContext()
+                .flatMap(ctx -> WebFluxSkyWalkingOperators.withSpanInScope(ctx, () -> {
```

Comment:
One question is `WebFluxSkyWalkingOperators#withSpanInScope` being called in the thread having to trace context, capture the snapshot, and continue this snapshot in the callback of another thread, right?

If I got this scenario correctly, I would recommend this API be renamed to `WebFluxSkyWalkingOperators#continueTracing`.